src/agent.py

```
@@ -176,30 +184,35 @@ async def chat(
         latest_message_id = latest_message.public_id if latest_message else None
         logger.debug(f"Latest user message ID: {latest_message_id}")
 
-        chat_history, _, _ = await history.get_summarized_history(
-            db_history, session_id, summary_type=history.SummaryType.SHORT
+        chat_history = await history.get_summarized_history(
+            db_history,
+            workspace_name,
+            session_name,
+            summary_type=history.SummaryType.SHORT,
         )
```

Pylint (3.3.7)

[error] No value for argument 'peer_name' in function call

(E1120)